### ISSUE TITLE
Plugin wont work with Grails 3.3.x And spring boot 1.5.3

### ISSUE THREAD

**Issue by snimavat** [sbglasius/external-config#13]
The contract for SpringApplicationRunListener interface has changed, 
`started()` method has been renamed to `starting() `

Out ExternalConfig run listener does not provide the implementation for the new method, resulting in 
Below exception while application is starting up

```
java.lang.AbstractMethodError
	at org.springframework.boot.SpringApplicationRunListeners.starting(SpringApplicationRunListeners.java:48)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:303)
```

It will prevent the app from starting.

**Comment by sbglasius** [comment]
I believe the latest snapshot version is working. Please test.

**Comment by snimavat** [comment]
Ah, i see, the change has been done, great...

**Comment by sbglasius** [comment]
I'm closing this issue again :)

**Comment by snimavat** [comment]
Are you sure the snapshot has been published to jfrog oss ? -- cant resolve 1.1.1-BUILD-SNAPSHOT' after adding maven { url "https://oss.jfrog.org/repo/" }

**Comment by sbglasius** [comment]
`Build successfully deployed. Browse it in Artifactory under http://oss.jfrog.org/webapp/builds/external-config/1498510429033`
